src/rgw/rgw_process.cc

```diff
@@ -327,18 +328,29 @@ int process_request(const RGWProcessEnv& penv,
     abort_early(s, NULL, -ERR_METHOD_NOT_ALLOWED, handler, yield);
     goto done;
   }
+  is_health_or_system_request = (
```

Comment:
nit:
```suggestion
const auto  is_health_or_system_request = (
```

Reply:
no i cannot mark it constant, as the variable is declared on line 310.
and i need to explicitly declare the variable above due to `goto` in the function :(

Reply:
well, the beuty of using `goto`...